src/ISADotnet/API/Update.fs

```diff
@@ -0,0 +1,117 @@
+﻿module Update
```

Comment:
Could you add this to a private module within namespace ISADotNet?

Also could you add some comments?